.github/workflows/main.yml

```diff
@@ -96,6 +102,7 @@ jobs:
       name: Checkout the repository
       with:
         submodules: recursive
+        persist-credentials: false
```

Comment:
Is there any way to make this a default? Otherwise, also needs to be documented.

Reply:
No, there isn't as far as I know. I'll document it.